### DISCUSSION TITLE
예약성공시 텔래그램 알림

### DISCUSSION THREAD

**Discussion by Happylula** [kimytsc/covid-rest-vaccine-macro#82] [Ideas]
멋진 프로그램 개발해주셔서 정말 감사합니다. 혹시 예약성공시 텔레그램 토큰과 아이디를 이용하여 예약이 완료되었다는 알람이 가게 할수있을까요??
코딩의 초보이고 한번도 해본적없는 자바스크립트다보니 유튜브나 구글서치를 해봐도 어디서부터 시작해야될지 막막하네요.. 
개발자님 혹시라도 시간이 나신다면 해당기능 부탁드리겠습니다..

**Comment by kimytsc** [comment]
텔레그램 메세지 보내는 소스는 이미 준비가 되어 있는 상태라 주말에 소스 정리해서 올려드리겠습니다.

**Reply by Happylula** [reply]
벌써 답글을 ㄷㄷ.. 감사합니다 올려주시는 코드랑 유튜브랑 서치등등 비교하면서 공부도 해봐야겠네요 ^^
